### PR TITLE
more fields

### PR DESCRIPTION
**Pull request by lepistone**

### CODE REVIEW

**Comment by lepistone** [comment]
I think we want to merge that quick. Then we'll switch to a shared branch as suggested by @sbidoul.

**Comment by lepistone** [comment]
btw good stuff! 👍

**Comment by damdam-s** [comment]
:+1:

File: account_financial_report_qweb/wizard/ledger_report_wizard.py [hunk 1]
```diff
@@ -11,3 +11,24 @@ class LedgerReportWizard(models.TransientModel):
...
+    date_from = fields.Date()
+    date_to = fields.Date()
+    target_move = fields.Selection([('posted', 'All Posted Entries'),
+                                    ('all', 'All Entries')],
+                                   string='Target Moves',
+                                   required=True,
+                                   default='posted')
+    account_ids = fields.Many2many(
+        comodel_name='account.account',
+        relation='account_account_ledger_rel',
```
Comment (pedrobaeza) [inline account_financial_report_qweb/wizard/ledger_report_wizard.py]: You don't need to specify relation attribute

**Comment by lepistone** [comment]
merged manually into `oca-sorrento/push-me`.

**Pull request closed without merging**